.dockerignore

```diff
@@ -0,0 +1,3 @@
+kubernetes/
+kubernetes.tar.gz
```

Comment:
Does this test actually download and build kubernetes?

Reply:
In prow job, it downloads (extract ci/lastest by default) and start multiple node cluster with GCE provider. It does not build kubernetes but provisioner is built under this directory, ignoring these files speeds up docker build. This is for performance optimization.

Reply:
It is weird at first glance, I added some comments.